demo/app.jsx

```diff
@@ -5,17 +5,47 @@ import {VictoryAxis} from "../src/index";
 class App extends React.Component {
   render() {
     const style = {
+      axis: {
+        marginTop: "50",
```

Comment:
radium pixel amounts can be passed in as numbers, no need to make everything a string